config-provisioning/src/main/java/com/yahoo/config/provision/NodeType.java

```diff
@@ -45,7 +45,7 @@
         this.description = description;
     }
 
-    public boolean isHost() {
+    public boolean isDockerHost() {
```

Comment:
Why not?

Reply:
Breaks the config server/config model API - in theory, did not look at usages.
I still want to do it but in a separate PR.